unliftio/src/UnliftIO/Async.hs

```diff
@@ -622,9 +701,9 @@ runFlat (FlatApp (FlatPure x)) = pure x
 
 -- Start off with all exceptions masked so we can install proper cleanup.
 runFlat f0 = E.uninterruptibleMask $ \restore -> do
-  -- How many threads have terminated? We need to ensure we kill all
+  -- How many threads have started? We need to ensure we kill all
```

Comment:
This change in wording surprises me. Is this a change in the code that I didn't notice, or a mistake in the original comment?

Reply:
I improved the comment with

```haskell
   -- How many threads have been spawned and finished their task? We need to
   -- ensure we kill all child threads and wait for them to die.
```